en/presenters.texy

```diff
@@ -685,9 +685,100 @@ You don't have to mark subcomponents as persistent, they are persistent automati
 
 
 Where can i get some components?
---------------------------------
+================================
 
 On page [Add-ons, plugins and components |addons:] you can find some open-source components that were made and shared by community of Nette Framework. Nette Foundation is not liable for them.
 
+
+
+Configuration
+*************
+
+We can setup many application features using configuration file. We write this configuration under the section `application` in the config.neon.
```

Comment:
No, tady záleží, co přesně chceš říct a tak :) Osobně by se mi pozdávalo víc něco jako: "We can modify application settings in the `config.neon` configuration file, under `application` section."